setup.py

```diff
@@ -23,25 +23,9 @@
 version = "4.3.0.dev0"
 
 import os
-import sys
 from setuptools import setup, find_packages
 
-if sys.version_info < (2, 6):
-    print("This version of ZODB requires Python 2.6 or higher")
-    sys.exit(0)
-
-if (3,) < sys.version_info < (3, 2):
-    print("This version of ZODB requires Python 3.2 or higher")
-    sys.exit(0)
-
-
-# The (non-obvious!) choices for the Trove Development Status line:
-# Development Status :: 5 - Production/Stable
-# Development Status :: 4 - Beta
-# Development Status :: 3 - Alpha
-
 classifiers = """\
-Development Status :: 4 - Beta
```

Comment:
I just tried to find a definition of this field and couldn't find one.  Maybe I should have searched harder. <shrug>
Given the ambiguity. all the more reason to avoid it.  I certainly thought it related to releases, but I was probably wrong.

FWIW I typically interpret 0.x releases as less than mature.  People (myself included) often are too lazy to make 1.x releases when they deem something mature.  These people aren't going to make releases to update the trove status.  BTW the trove status: "Development Status :: 7 - Inactive" makes me laugh. :)